packages/beacon-node/src/chain/seenCache/seenGossipBlockInput.ts

```diff
@@ -361,111 +361,116 @@ export class SeenGossipBlockInput {
     //   blockInputMeta: {pending: GossipedInputType.block, haveBlobs: blobsCache.size, expectedBlobs: null},
     // };
   }
+}
 
-  private hasSampledDataColumns(dataColumnCache: DataColumnsCacheMap): boolean {
-    return (
-      dataColumnCache.size >= this.custodyConfig.sampledColumns.length &&
-      this.custodyConfig.sampledColumns.reduce((acc, columnIndex) => acc && dataColumnCache.has(columnIndex), true)
-    );
+function hasSampledDataColumns(custodyConfig: CustodyConfig, dataColumnCache: DataColumnsCacheMap): boolean {
+  return (
+    dataColumnCache.size >= custodyConfig.sampledColumns.length &&
+    custodyConfig.sampledColumns.reduce((acc, columnIndex) => acc && dataColumnCache.has(columnIndex), true)
+  );
+}
+
+export async function getDataColumnsFromExecution(
```

Comment:
Lets move these to utils folder.  Not sure if `src/util/dataColumns.ts` is the right place.  What do you think?

Reply:
I think for now dataColumns is the place to put it, but maybe a future pr could make a `util/dataColumns` folder, and have subfiles for `custodyConfig`, `recovery`/`reconstruction`, etc